lib/gitlab/ci/templates/pipeline/TerraformPipeline.yml

```diff
@@ -0,0 +1,73 @@
+variables:
+  TF_ROOT: ${CI_PROJECT_DIR}
+  PLAN_FILE_NAME: planfile
+  TF_STATE_NAME: default
+  TF_CACHE_KEY: default
+  # If your terraform files are in a subdirectory, set TF_ROOT accordingly. For example:
+  # TF_ROOT: terraform/production
+
+image:
+  name: hashicorp/terraform:light
```

Comment:
Can you add an optional prefix here? CI pipelines hit a pull limit pretty quickly unless we pull from a cache